build.rs

```diff
@@ -213,45 +214,70 @@ fn main() {
     }
 }
 
-fn make_zlib(compiler: &cc::Tool, src_dir: &path::Path, out_dir: &path::Path) {
-    let src_dir = src_dir.join("zlib");
+fn make_zlib(compiler: &cc::Tool, src_dir: &path::Path, _: &path::Path) {
     // lock README such that if two crates are trying to compile
     // this at the same time (eg libbpf-rs libbpf-cargo)
     // they wont trample each other
-    let file = std::fs::File::open(src_dir.join("README")).unwrap();
+    let file = std::fs::File::open(src_dir.join("README.md")).unwrap();
```

Comment:
Why is that being changed? The file seems to be called `README`.

Reply:
> Why is that being changed? The file seems to be called .`README`

Oh, that was my problem, the locked file was `libbpf-sys/README.md` instead of `libbpf-sys/zlib/README` I'll fix it